spec/models/course_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+require "rspec/collection_matchers"
+require "shoulda-matchers"
+
+RSpec.describe Course, type: :model do
+  let(:course) {FactoryBot.create :course}
+  subject {course}
+
+  it "is a Course" do
+    is_expected.to be_a_kind_of Course
+  end
+
+  it "is valid with all correct informations" do
+    is_expected.to be_valid
+  end
+
+  describe "#name" do
```

Comment:
Phân biệt describe vs context ? trường hợp nào dùng cái nào?

Reply:
Về chức năng thì hai phương thức này giống nhau là điều dùng để nhóm các tests.
- describe dùng để nhóm các test cùng một chức năng, describe riêng biệt cho các thuộc tính cần validation.
- context dùng để nhóm các test cùng một chức năng và có cùng 1 trạng thái, vd: context "when course is started" { }

Reply:
Về chức năng thì hai phương thức này giống nhau là điều dùng để nhóm các tests.
- describe dùng để nhóm các test cùng một chức năng, describe riêng biệt cho các thuộc tính cần validation.
- context dùng để nhóm các test cùng một chức năng và có cùng 1 trạng thái, vd: context "when course is started" { }